File: src/com/bonplan/entities/Produit.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.bonplan.entities;


/**
 *
 * @author bouyo
 */
public class Produit {

    private int idProduit;
    private User idOwer;
    private String categorieProduit;
    private String nomProduit;
    private String descriptionProduit;
    private String photoProduit;
    private String videoProduit;
    private float prixProduit;
    private int stockProduit;

    public Produit() {
    }

   
    
     public Produit(int idProduit ,String nomProduit , String descriptionProduit, String photoProduit,  float prixProduit, int stockProduit,  String categorieProduit
) {        
         this.idProduit = idProduit;
         this.nomProduit = nomProduit;

        this.categorieProduit = categorieProduit;
        this.descriptionProduit = descriptionProduit;
        this.photoProduit = photoProduit;
        this.prixProduit = prixProduit;
        this.stockProduit = stockProduit;
    }
     public Produit(String categorieProduit, User idOwer, String descriptionProduit, String photoProduit,  float prixProduit, int stockProduit,  String nomProduit
) {        
        this.idOwer =idOwer;
        this.nomProduit = nomProduit;
         this.categorieProduit = categorieProduit;
        this.descriptionProduit = descriptionProduit;
        this.photoProduit = photoProduit;
        this.prixProduit = prixProduit;
        this.stockProduit = stockProduit;
    }
    public Produit( String categorieProduit, String descriptionProduit, String photoProduit, String videoProduit, float prixProduit, int stockProduit,  String nomProduit
) {        this.nomProduit = nomProduit;

        this.categorieProduit = categorieProduit;
        this.descriptionProduit = descriptionProduit;
        this.photoProduit = photoProduit;
        this.videoProduit = videoProduit;
        this.prixProduit = prixProduit;
        this.stockProduit = stockProduit;
    }
    public Produit( String categorieProduit, String descriptionProduit, String photoProduit,  float prixProduit, int stockProduit,  String nomProduit
) {        this.nomProduit = nomProduit;

        this.categorieProduit = categorieProduit;
        this.descriptionProduit = descriptionProduit;
        this.photoProduit = photoProduit;
        this.prixProduit = prixProduit;
        this.stockProduit = stockProduit;
    }
    public Produit(String categorieProduit, String descriptionProduit, String photoProduit, String videoProduit) {
        this.categorieProduit = categorieProduit;
        this.descriptionProduit = descriptionProduit;
        this.photoProduit = photoProduit;
        this.videoProduit = videoProduit;
    }
 public Produit(String descriptionProduit, String photoProduit, String videoProduit) {
        this.descriptionProduit = descriptionProduit;
        this.photoProduit = photoProduit;
        this.videoProduit = videoProduit;
    }
   

    public Produit(String nom_produit, String description, String photo, float prix_produit, int stock_produit, String categorie_produit, String video) {
 this.nomProduit = nom_produit;
        this.descriptionProduit = description;
        this.photoProduit = photo;
        this.categorieProduit = categorie_produit;
        this.prixProduit = prix_produit;
        this.stockProduit = stock_produit;
        this.videoProduit= video;
    }

    public Produit(int id_produit, String nom_produit, String description, String photo, float prix_produit, int stock_produit, String categorie_produit, String video) {
        this.idProduit= id_produit;
        this.nomProduit = nom_produit;
        this.descriptionProduit = description;
        this.photoProduit = photo;
        this.categorieProduit = categorie_produit;
        this.prixProduit = prix_produit;
        this.stockProduit = stock_produit;
        this.videoProduit= video;    }

    public Produit(int i) {
        this.idProduit= i;
    }

   


  

   

  

    public int getIdProduit() {
        return idProduit;
    }

    public void setIdProduit(int idProduit) {
        this.idProduit = idProduit;
    }

    public String getCategorieProduit() {
        return categorieProduit;
    }

    public void setCategorieProduit(String categorieProduit) {
        this.categorieProduit = categorieProduit;
    }

    public String getNomProduit() {
        return nomProduit;
    }

    public void setNomProduit(String nomProduit) {
        this.nomProduit = nomProduit;
    }

    public String getDescriptionProduit() {
        return descriptionProduit;
    }

    public void setDescriptionProduit(String descriptionProduit) {
        this.descriptionProduit = descriptionProduit;
    }

    public String getPhotoProduit() {
        return photoProduit;
    }

    public void setPhotoProduit(String photoProduit) {
        this.photoProduit = photoProduit;
    }

    public String getVideoProduit() {
        return videoProduit;
    }

    public void setVideoProduit(String videoProduit) {
        this.videoProduit = videoProduit;
    }

    public float getPrixProduit() {
        return prixProduit;
    }

    public void setPrixProduit(float prixProduit) {
        this.prixProduit = prixProduit;
    }

    public int getStockProduit() {
        return stockProduit;
    }

    public void setStockProduit(int stockProduit) {
        this.stockProduit = stockProduit;
    }

    public User getIdOwer() {
        return idOwer;
    }

    public void setIdOwer(User idOwer) {
        this.idOwer = idOwer;
    }


    
    

}

File: web/ProduitBundle/Entity/Produit.php
<?php

namespace PIDEV\ProduitBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert ;


/**
 * Produit
 *
 * @ORM\Table(name="produit")
 * @ORM\Entity(repositoryClass="PIDEV\ProduitBundle\Repository\ProduitRepository")
 */
class Produit
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_produit", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id_produit;
    /**
     * @var
     * @ORM\ManyToOne(targetEntity="PIDEV\UserBundle\Entity\User")
     * @ORM\JoinColumn(name="id_owner",referencedColumnName="id",nullable=true, onDelete="SET NULL")
     */
    private $id_owner;

    /**
     * @var string
     *
     * @ORM\Column(name="categorie_produit", type="string", length=255)
     */
    private $categorie_produit;


    /**
     * @var string
     *
     * @ORM\Column(name="nom_produit", type="string", length=255)
     */
    private $nom_produit;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="string", length=255)
     */
    private $description;

    /**
     * @var string
     *
     * @ORM\Column(name="photo", type="string", length=255,nullable=true)
     *  @Assert\NotBlank(message="Please, upload the product brochure")
     * @Assert\File(mimeTypes={ "image/jpg","image/png","image/jpeg" })
     */
    private $photo;

    /**
     * @var float
     *
     * @ORM\Column(name="prix_produit", type="float")
     */
    private $prix_produit;



    /**
     * @ORM\Column(name="stock_produit", type="integer" )
     */
    private $stock_produit;


    /**
     * @return int
     */
    public function getIdProduit()
    {
        return $this->id_produit;
    }

    /**
     * @param int $id_produit
     */
    public function setIdProduit($id_produit)
    {
        $this->id_produit = $id_produit;
    }

    /**
     * @return mixed
     */
    public function getIdOwner()
    {
        return $this->id_owner;
    }

    /**
     * @param mixed $id_owner
     */
    public function setIdOwner($id_owner)
    {
        $this->id_owner = $id_owner;
    }


    /**
     * @return string
     */
    public function getNomProduit()
    {
        return $this->nom_produit;
    }

    /**
     * @param string $nom_produit
     */
    public function setNomProduit($nom_produit)
    {
        $this->nom_produit = $nom_produit;
    }

    /**
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * @param string $description
     */
    public function setDescription($description)
    {
        $this->description = $description;
    }

    /**
     * @return string
     */
    public function getPhoto()
    {
        return $this->photo;
    }

    /**
     * @param string $photo
     */
    public function setPhoto($photo)
    {
        $this->photo = $photo;
    }


    /**
     * @return int
     */
    public function getPrixProduit()
    {
        return $this->prix_produit;
    }

    /**
     * @param int $prix_produit
     */
    public function setPrixProduit($prix_produit)
    {
        $this->prix_produit = $prix_produit;
    }

    /**
     * @return string
     */




    /**
     * @return mixed
     */
    public function getStockProduit()
    {
        return $this->stock_produit;
    }

    /**
     * @param mixed $stock_produit
     */
    public function setStockProduit($stock_produit)
    {
        $this->stock_produit = $stock_produit;
    }

    /**
     * @return string
     */
    public function getCategorieProduit()
    {
        return $this->categorie_produit;
    }

    /**
     * @param string $categorie_produit
     */
    public function setCategorieProduit($categorie_produit)
    {
        $this->categorie_produit = $categorie_produit;
    }





}

